c_src/membrane_rtmp_plugin/source/rtmp_source.c

```diff
@@ -3,7 +3,13 @@
 
 void handle_init_state(State *);
 
-UNIFEX_TERM native_create(UnifexEnv *env, char *url, char *timeout) {
+static int interrupt_cb(void *ctx) {
```

Comment:
Full name perhaps?
```suggestion
static int interrupt_callback(void *ctx) {
```

Reply:
I believe that is the ffmpeg's convention. Not sure if we should follow it or not though.

Reply:
Let's call a spade a spade - it's not a result of following the convention, it's just a copy-paste from ffmpeg's example :upside_down_face: And we definitely should follow more Elixir-like naming convention :wink: